onnxruntime/python/tools/microbench/benchmark.py

```diff
@@ -8,14 +8,15 @@
 
 def numpy_type(torch_type):
     type_map = {torch.float32: numpy.float32,
-                torch.float16: numpy.float16}
+                torch.float16: numpy.float16,
+                torch.int32: numpy.int32}
     return type_map[torch_type]
 
 
 def add_arguments(parser: ArgumentParser):
-    parser.add_argument("--provider", required=False, type=str, default="rocm", help="Execution provider to use")
-    parser.add_argument("--precision", required=False, type=str, default="fp16", help="Number format to use")
-    parser.add_argument('--profiling', type=bool, default=False, help='If enable profiling')
+    parser.add_argument("--provider", required=False, type=str, choices=["rocm", "cuda"], default="rocm", help="Execution provider to use")
```

Comment:
getting better, but the message is still confusing

/bert_ort/ettao/conda/py36/lib/python3.6/site-packages/onnxruntime/capi/onnxruntime_inference_collection.py:56: UserWarning: Specified provider 'ROCMExecutionProvider' is not in available provider names.Available providers: 'CUDAExecutionProvider, CPUExecutionProvider'

would be even better to print choices instead

Reply:
Made some changes; now the CUDA/ROCM EP will be automatically selected by default.